### PR TITLE
Korean translation of the login theme

### PR DESCRIPTION
**Pull request by javapark**

### CODE REVIEW

**Comment by ahus1** [comment]
Thank you for this translation. For this PR to be merged, we'll need a native speaker to confirm the translations you've prepared. 

Can you please ask for example another co-worker or community member to review your translations and comment here so that we know they have been reviewed? Thanks!

File: themes/src/main/resources-community/theme/base/login/messages/messages_ko.properties
Comment (loveis) [inline themes/src/main/resources-community/theme/base/login/messages/messages_ko.properties:1]: 그냥 사용해도 될듯 합니다.

**Comment by loveis** [comment]
그냥 사용해도 될듯 합니다.

**Comment by ahus1** [comment]
@loveis - I see your previous comment. Can you please write here in English if you reviewed the translation and that it look ok for you? Thanks!

**Comment by loveis** [comment]
Finally, we have a Korean property, and it's very nice.

**Comment by han-chunsik** [comment]
좋습니다! good!

**Review by ahus1 — APPROVED** [review]
Thank you for this contribution and also the review from the community!

**Pull request merged**